rmw_implementation/src/functions.cpp

```diff
@@ -22,7 +22,7 @@
 #include <fstream>
 #include <sstream>
 
-#include "c_utilities/types.h"
+#include "rcutils/types.h"
```

Comment:
including "rcutils/types/string_array.h" , so it stays consistent with the other DDS impl.